testing/test_cluster.py

```diff
@@ -38,6 +39,27 @@ def make_cluster(self, num_nodes):
                                                             self.ramcloud_network)
         self.rc_client.connect(external_storage, 'main')
 
+    def simple_recovery(self, use_force):
```

Comment:
`use_force` fully determines `sigspec` below, why not just pass `sigspec` directly into `simple_recovery`? Alternatively the entire kill command could be passed in which could be used to kill coordinators as well as servers.

Reply:
Going with the latter